src/core/server/http/integration_tests/request.test.ts

```diff
@@ -13,11 +13,9 @@ jest.mock('uuid', () => ({
 import supertest from 'supertest';
 import { loggingSystemMock } from '@kbn/core-logging-server-mocks';
 import { executionContextServiceMock } from '@kbn/core-execution-context-server-mocks';
-
-import { HttpService } from '../http_service';
-
-import { contextServiceMock } from '../../context/context_service.mock';
-import { createHttpServer } from '../test_utils';
+import { contextServiceMock } from '@kbn/core-http-context-server-mocks';
+import { HttpService } from '@kbn/core-http-server-internal';
```

Comment:
I don't want to be annoying with the NITs 😅 .
There's a bunch more places where `HttpService` is only used as a type.

Reply:
Yea, now it one of the best opportunity to fix these NITs, so don't hesitate to add more :)